ocs_ci/helpers/osd_resize.py

```diff
@@ -192,20 +192,40 @@ def check_storage_size_is_reflected(expected_storage_size):
             f"The PV sizes are not equal to the expected storage size {expected_storage_size_in_gb}"
         )
 
+    if expected_ceph_capacity:
+        check_ceph_capacity_increased(expected_ceph_capacity)
+
+
+def check_ceph_capacity_increased(
+    expected_ceph_capacity,
+    tolerance_percent=1.0,
```

Comment:
can you explain in why 1.0 in docstring?

Reply:
The expected and actual Ceph capacity should ideally be the same. However, in practice, minor mismatches can occur due to the way Ceph calculates capacity in bytes, which may introduce minor differences. Additionally, rounding issues can arise when converting between different size units (e.g., TiB to GiB), which can slightly affect the calculated values. To account for these variations, we apply a small tolerance in the validation.
A 1.0% difference seems reasonable, but we can consider a higher threshold if needed.